bob-common/src/metrics/exporters/graphite_exporter/mod.rs

```diff
@@ -89,82 +59,13 @@ impl GraphiteBuilder {
         self
     }
 
-    pub(crate) fn build(self) -> GraphiteRecorder {
-        let (tx, rx) = channel(BUFFER_SIZE);
-        let recorder = GraphiteRecorder { tx };
-        tokio::spawn(send_metrics(rx, self.address, self.interval, self.prefix));
-        recorder
-    }
-}
-
-impl GraphiteRecorder {
-    fn push_metric(&self, m: Metric) {
-        if let Err(e) = self.tx.try_send(m) {
-            error!(
-                "Can't send metric to thread, which processing metrics: {}",
-                e
-            );
-        }
-    }
-}
-
-// TODO: impl recorder in proper way
-impl Recorder for GraphiteRecorder {
-    fn register_gauge(
-        &self,
-        key: &Key,
-        _unit: Option<metrics::Unit>,
-        _description: Option<&'static str>,
-    ) {
-        self.update_gauge(key, GaugeValue::Absolute(0f64));
-    }
-
-    fn register_counter(
-        &self,
-        key: &Key,
-        _unit: Option<metrics::Unit>,
-        _description: Option<&'static str>,
-    ) {
-        self.increment_counter(key, 0);
-    }
-
-    fn register_histogram(
-        &self,
-        key: &Key,
-        _unit: Option<metrics::Unit>,
-        _description: Option<&'static str>,
-    ) {
-        self.record_histogram(key, 0f64);
-    }
-
-    fn increment_counter(&self, key: &Key, value: u64) {
-        self.push_metric(Metric::Counter(MetricInner::new(
-            key.name().to_owned(),
-            value,
-            -1,
-        )));
-    }
-
-    #[allow(clippy::cast_sign_loss)]
-    fn update_gauge(&self, key: &Key, value: GaugeValue) {
-        let val = if let GaugeValue::Absolute(val) = value {
-            val
-        } else {
-            error!("Diffs are not supported at the moment");
-            return;
-        };
-        self.push_metric(Metric::Gauge(MetricInner::new(
-            key.name().to_owned(),
-            val as u64,
-            -1,
-        )));
-    }
-
-    fn record_histogram(&self, key: &Key, value: f64) {
-        self.push_metric(Metric::Time(MetricInner::new(
-            key.name().to_owned(),
-            value as u64,
-            -1,
-        )));
+    // TODO move recorder and snapshot to other place?
```

Comment:
What do you mean? Move in project hierarchy?

Reply:
Forgot to remove this comment after refactoring, it's already done (different mod)